src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-bar__form form-search-bar--big"
+      action="#"
+      data-qa="big"
+      method="post">
```

Comment:
the same. [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) rule number 5